en/docs/install-and-setup/setup/api-controller/managing-common-api-policies/importing-common-api-policies-via-dev-first-approach.md

```diff
@@ -0,0 +1,84 @@
+# ImportingCommon API Policy Via Dev First Approach
```

Comment:
We do not need a DevFirst doc for any of the policies, because we do not recommend the customers to create policy artifacts from the scratch. Let's remove this doc.

Reply:
Fixed by [6465f5a](https://github.com/wso2/docs-apim/pull/6103/commits/6465f5a2f5f93d8ca2645700a95ab612cd0d25ed)